### DISCUSSION TITLE
How can I hide unwanted "browsers" in v16?

### DISCUSSION THREAD

**Discussion by MTCoster** [will-stone/browserosaurus#461] [Q&A] [answered]
Up to v15, I was able to hide any browsers I don't want to appear in the popup in the settings. However, I can't seem to find an option for this in v16!

For instance, Browserosaurus discovers Spotify and Zoom as browsers which may be very useful for some people! However, I so rarely click on links for those apps I just don't need them to appear in the list since the web UIs have links to open the native apps in those rare situations.

![Screenshot 2021-11-29 at 15 50 36](https://user-images.githubusercontent.com/2152582/143899461-9ee411f0-e4da-4db0-8462-527c8075d6ba.png)

**Comment by will-stone** [comment, marked as answer]
Covered in the release notes: https://github.com/will-stone/browserosaurus/releases/tag/v16.0.6

I like the simplicity of not needing to do this (and maintain the code for this) any more. Keep your less used apps at the bottom, voilà!

**Reply by nicoblabla** [reply]
Hey, thanks for this really great app.

I also really like the proposed idea.
If I were to implement this feature, is there a chance that the pull request would be merged?

**Reply by will-stone** [reply]
Hey @nicoblabla 👋 Thank you for the offer but I wouldn’t be adding this to the app. The best solution is no code at all: move your unused apps to the bottom and shrink the picker. 

There are issues marked as “help wanted” if you’d like to contribute to the project.

Cheers 😄

**Reply by mikaelf** [reply]
The feature to "Drag to sort" the list of browsers is not easily discoverable. I was about to file a bug that browsers that I had assigned Keys to wasn't in the top of the list, only to find info about the ability to sort the browsers. Perhaps there could be instructions in settings Preferences -> Apps?

**Reply by will-stone** [reply]
Could do. A little note in there maybe. Give it a go @mikaelf, in a PR, and I'll happily review it.

**Reply by ProLoser** [reply]
You can always leave the fork open regardless of the review @nicoblabla @mikaelf

**Reply by nZeus** [reply]
Hi. Thank you for working on this nice tool!
I hope you'll consider implementing the "hide" option eventually. I would like to hide a few apps (Safari/Teams/Slack) as for me it doesn't make any sense to have them in the list
